### DISCUSSION TITLE
Duda tiempos de ejecución

### DISCUSSION THREAD

**Discussion by diegoftpxd** [IIC2133-PUC/2024-2#160] [Tarea 2] [answered]
Hola, tengo una duda con respecto a los tiempos límite en la ejecución. Me pasa que los test de mayor dificultad los paso pero con un tiempo de entre 4 y 6 segundos dependiendo de la ejecución. Entonces mi duda va a que como es un tiempo que depende del computador que lo corra ¿Como puedo comprobar que el tiempo está dentro de lo esperado? y ¿más o menos en cuanto tiempo es recomendable para estar seguro de que cumplo el requerimiento de tiempo?

**Comment by sebacarrascop** [comment, marked as answer]
hola!
Para verificar el tiempo de ejecución puedes usar el comando time antes de tu ejecutable.
`time ./exec`
Dado que hartas personas han tenido problemas con correr los hard bajo los 5 segundos, bajaremos levemente su complejidad en los tests privados, y los ejecutaremos por 10 segundos, ya que probablemente la complejidad esta bien, pero un factor constante debe estar afectando su respuesta con respecto a nuestro código solución con el cual probamos los tests.
Saludos!
